addons/html_builder/static/src/interactions/google_map.edit.js

```diff
@@ -3,83 +3,78 @@
 import { GoogleMap } from "@website/snippets/s_google_map/google_map";
 import { registry } from "@web/core/registry";
 
-const GoogleMapsEdit = (I) =>
-    class extends I {
-        setup() {
-            super.setup();
-            this.canSpecifyKey = true;
-            this.websiteEditService = this.services.website_edit;
-            this.websiteMapService = this.services.website_map;
-        }
+class GoogleMapsEdit extends GoogleMap {
```

Comment:
Since it starts from the base `GoogleMap` interaction anyway, why can't it be done with the mixin approach ?
I know this is convoluted, but it is about being able to patch for edit mode at distinct inheritance levels - we therefore tried to do it that way on all edit interactions, to avoid showing the `extends` way to other developers.